lib/utils.js

```diff
@@ -147,6 +147,27 @@ function generateLogPrefix(obj, sessionId = null) {
   );
 }
 
+/**
+ * Converts the given object to an integer number if possible
+ *
+ * @param {any} value to be converted
+ * @returns {number?}
+ */
+function convertToInt(value) {
+  switch (typeof value) {
+    case 'number':
+      return value;
```

Comment:
typeof NaN === 'number' in javascript ;)

Reply:
Thanks, I'll fix it and add a test

Reply:
fixed https://github.com/appium/appium-chromedriver/pull/291/commits/6505403a5cb7c03fac55ea2e6f73b3de17108191